### DISCUSSION TITLE
webassembly/webgl, does co_yield work?

### DISCUSSION THREAD

**Discussion by unicomp21** [mosra/magnum#573] [Q&A] [answered]
i know co_wait doesn't work, but wondering if co_yield works? @mosra

**Comment by mosra** [comment]
I have no idea, I don't use coroutines myself. This question is probably better asked on the Emscripten project itself, it doesn't really have to do anything with Magnum.

**Comment by Squareys** [comment, marked as answer]
@unicomp21 Since Magnum uses emscripten, the answer is the same as for your issue on the emscripten repository 😌

https://github.com/emscripten-core/emscripten/issues/10991
